addons/base/views.py

```diff
@@ -664,6 +666,9 @@ def addon_view_or_download_file(auth, path, provider, **kwargs):
             'Location': file_node.generate_waterbutler_url(**dict(extras, direct=None, version=version.identifier, _internal=extras.get('mode') == 'render'))
         }))
 
+    if action == 'view' and provider == 'osfstorage':
+        update_analytics(node_addon.owner, file_node._id, int(version.identifier) - 1, download=False)
```

Comment:
I think we'll need to remove this when https://openscience.atlassian.net/browse/PLAT-218 after implemented.